src/components/helpers/sortPeople.ts

```diff
@@ -0,0 +1,25 @@
+import { Person } from '../../types';
+import { SortType } from '../enums/SortType';
+
+export const sortPeople = (
+  people: Person[],
+  sortType: string | null,
+  order: string | null,
+) => {
```

Comment:
You should create a copy of the `people` array before sorting to avoid mutating the original array. You can use the spread operator (`...`) to create a shallow copy: `const sortedPeople = [...people];` 🚀